app/src/main/java/io/appium/uiautomator2/handler/SendKeysToElement.java

```diff
@@ -45,7 +45,8 @@ public SendKeysToElement(String mappedUri) {
     }
 
     private static boolean isTextFieldClear(AndroidElement element) throws UiObjectNotFoundException {
-        return element.getText() == null || element.getText().isEmpty();
```

Comment:
actually the previous one was also fine ;)

Reply:
Yes, just two calls to `element.getText()`.